src/biokbase/narrative/app_util.py

```diff
@@ -561,13 +585,13 @@ def validate_param_value(param, value, workspace):
                 ws_ref = f"{info[6]}/{info[0]}/{info[4]}"
             type_ok = False
             for t in param["allowed_types"]:
-                if re.match(t, info[2]):
+                if t == "*" or re.match(t, info[2]):
```

Comment:
This is the whole fix right here.

Previously, we'd be matching against types like this:
`re.match("KBaseGenomes.Genome", "SomeOther.Type")`

Now there's the supported option of:
`re.match("*", "SomeOther.Type")`
having that `*` as the first character makes it think you're starting with matching a wildcard. Really, we don't care what type of object you're trying to match with, so skip it in that case.